group_members.go

```diff
@@ -201,6 +201,24 @@ func (s *GroupMembersService) ShareWithGroup(gid interface{}, opt *ShareWithGrou
 	return g, resp, err
 }
 
+// RemoveShareWithGroup allows to unshare a group from a group.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/groups.html#delete-link-sharing-group-with-another-group
+func (s *GroupMembersService) RemoveShareWithGroup(gid interface{}, groupID int, options ...RequestOptionFunc) (*Response, error) {
```

Comment:
`RemoveShareWithGroup` or `DeleteShareWithGroup` ?
On the project side, its implemented in the `Project` struct instead of `ProjectMembers` and is named `DeleteSharedProjectFromGroup`.

Reply:
Project side all are named `Delete*`, however here we have `RemoveGroupMember`. 🤔

Reply:
I prefer to follow the docs (and their layout) as much as possible. So that would mean to use the name `DeleteShareWithGroup`. As for the `Project` vs `ProjectMembers`, I think this has changes over time and the docs layout (and API) was different when this was created.

While I'm all for consistency, in this case I would not try to fix it as it probably causes more harm then good. So let's keep it here, but update the name. Thanks @horjulf!

Reply:
Sounds good, updated.